modes/delete.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {

```

Comment:
This should be in a separate PR

Reply:
That was an accident, I tried to get rid of it by cancelling my other PR but it didn't work, ill fix it now

Reply:
You can create a second branch to separate the changes

Reply:
I'll just delete it for now